### PR TITLE
Prof dashboard

### PR DESCRIPTION
**Pull request by faizulx**
Resolved merge conflicts.

### CODE REVIEW

**Review by pdx-cs-lbryan — CHANGES_REQUESTED** [review]
1.  please remove any file that's not in master unless you're adding it for your code (ex: db_form.py, db_test.db)
2. may need to look into licenses for some of the libraries used. We can talk to the group about this.
3. please leave the run command in app.py as it is in master
4. Should selecting a term currently display only the students/teams in that term? Right now it's showing all of then.
5. We'll probably want to work on styling the menu bar since it's getting a little thick, but that can come later.
6. Seems 'Edit' is bugged.
7. There are some unresolved merge conflicts (ex: queryrunner, requirements)

File: .vscode/launch.json [hunk 1]
```diff
@@ -0,0 +1,70 @@
+{
```
Comment (J8cob1) [inline .vscode/launch.json:1]: This looks like an ide specific file. Should we have this here?

File: templates/form/response.html [hunk 2]
```diff
@@ -1,7 +1,11 @@
 <!DOCTYPE html>
 <head>
     <title>
+<<<<<<< HEAD
+        <Title>Review Submitted</Title>
+=======
         Review Submitted
+>>>>>>> master
```
Comment (J8cob1) [inline templates/form/response.html:8]: Unresolved

File: templates/form/review.html [hunk 3]
```diff
@@ -1,4 +1,16 @@
...
+<nav>
+  {% if g.user %} -- needs log in
+  <h1>Welcome to the Midterm 360 Review, {{ g.user['username'] }}</h1>
+  {% endif %}
+  <h1>Welcome to the Midterm 360 Review</h1>
+  -->
+=======
 
+>>>>>>> master
   {% extends "layout.html" %}
```
Comment (J8cob1) [inline templates/form/review.html:14]: Unresolved merge conflict

File: app.py [hunk 4]
```diff
@@ -35,30 +35,10 @@
...
-                view_func=AddTeam.as_view('addTeam'),
-                methods=['GET', 'POST'])
-
-app.add_url_rule('/removeStudent/',
-                view_func=RemoveStudent.as_view('removeStudent'),
-                methods=['GET', 'POST'])
-
-app.add_url_rule('/removeTeam/',
-                view_func=RemoveTeam.as_view('removeTeam'),
-                methods=['GET', 'POST'])
```
Comment (J8cob1) [inline app.py:60]: Do we need to nuke all of this code?

File: capstone360/__init__.py [hunk 5]
```diff
@@ -0,0 +1,20 @@
+# main file for server management
```
Comment (J8cob1) [inline capstone360/__init__.py:1]: Do we need this file here right now? Didn't we remove this? I think we may need to move everything back into capstone360, but for now, no.

File: app.py [hunk 6]
```diff
@@ -35,30 +35,10 @@
...
+    form = dashboard.Form()
+    team_names = dashboard.get_teams()
+    member_names = dashboard.get_members()
+    return render_template('dashboard.html', lists=lists, sessions=sessions, team_names=team_names, member_names=member_names, form=form)
```
Comment (J8cob1) [inline app.py:41]: Can we put all of this into a MethodView class (like the ones in removeClass.py) and use an app.add_url_rule(...) command in app.py to add it to the app?

**Review by J8cob1 — CHANGES_REQUESTED** [review]

**Pull request closed without merging**